ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -195,12 +152,9 @@ public void onRequestPermissionsResult(
     final int requestCode,
     final String[] permissions,
     final int[] grantResults) {
-    mPermissionsCallback = new Callback() {
-      @Override
-      public void invoke(Object... args) {
-        if (mPermissionListener != null && mPermissionListener.onRequestPermissionsResult(requestCode, permissions, grantResults)) {
-          mPermissionListener = null;
-        }
+    mPermissionsCallback = args -> {
```

Comment:
Could you change this to not use a lambda expression? We are still supporting older versions of Java that don't have them.

Reply:
Yup definitely. Curious question, what version of Java do you back support and is there a specific reason why? Just wanting to learn from it.